### ISSUE TITLE
Level Download Failed

### ISSUE THREAD

**Issue by GDMpyre** [Cvolton/GMDprivateServer#889]
Whenever i try to download a level it gives me an error, everything else works fine.

This only happens with levels that existed before switching GDPS hosts, since my GDPS used to be hosted my FHGDPS, but since it shut down I moved it to CrystalCloud. I created a test level after switching and it loads just fine.

Every single file (except the config folder) has CHMOD set to 777 and there are no errors with the connection since accounts, leaderboards and levels (excluding downloading them) do work properly.

**Comment by Cvolton** [comment]
to be clear, can you confirm that you carried over the data folder from the old host?

**Comment by MegaSa1nt** [comment]
yes, probably old levels are not in data folder

**Comment by GDMpyre** [comment]
> to be clear, can you confirm that you carried over the data folder from the old host?

yeah, the levels are in data/levels, they do appear in the level menu, but can't download them. And there are over 500 levels so I would prefer being able to fix this issue instead of having to start all over.

**Comment by MegaSa1nt** [comment]
they appear in level menu because they are in mysql database

**Comment by IIBriXXII** [comment]
I ran into this exact Issue too, everything works except downloading the old levels. The Levels are in the data folder and all levels are set to 777 (the levels folder is also at 777)

https://imgur.com/k7G2KeW
